cmd/gather/cluster/cluster.go

```diff
@@ -295,11 +322,28 @@ func (c *Cluster) processOTELCollector(otelCol *otelv1beta1.OpenTelemetryCollect
 	return nil
 }
 
+func (c *Cluster) processOTELTargetAllocator(ta *otelv1alpha1.TargetAllocator) error {
+	log.Printf("Processing TargetAllocator %s/%s", ta.Namespace, ta.Name)
+	folder, err := createOTELFolder(c.config.CollectionDir, ta.ObjectMeta)
```

Comment:
Won't this cause conflicts if you have a OpenTelemetryCollector and TargetAllocator with the same name?